client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
`quiet` and `noInfo` are only used in webpack-dev-middleware. If `noInfo: true`, the console will only show user errors. If `quiet: true`, the console won't show anything.

Maybe it's too big of a hack to reuse this for client errors though. I can imagine most people use `noInfo` or `quiet` _because_ they can also see it in their browser.

If we'd make a separate option though, I would prefer using a "log level". Maybe `clientLogLevel`, which takes `null` (no logs), `"error"`, `"warn"`, `"info"` (default)?